packages/dds/tree/src/test/rebaserAxiomaticTests.ts

```diff
@@ -380,6 +654,196 @@ function verifyComposeAssociativity<TChangeset>(
 	assertDeepEqual(rightPartialCompositions.at(-1), singlyComposed);
 }
 
+function verifyComposeWithInverseEqualsEmpty<TChangeset>(
+	edits: TaggedChange<TChangeset>[],
+	fieldRebaser: BoundFieldChangeRebaser<TChangeset>,
+) {
+	const metadata = defaultRevisionMetadataFromChanges(edits);
+
+	for (const edit of edits) {
+		const change = tagChange(edit.change, edit.revision);
+		const changeset = fieldRebaser.compose(
+			change,
+			tagChange(fieldRebaser.invert(change), change.revision),
+			metadata,
+		);
+		const actualChange = makeAnonChange(changeset);
+		assert(fieldRebaser.isEmpty !== undefined);
+		assert(fieldRebaser.isEmpty(actualChange.change));
+	}
+}
+
+function verifyComposeWithEmptyIsNoOp<TChangeset>(
+	edits: TaggedChange<TChangeset>[],
```

Comment:
It seems like this should take a single edit instead of an array.

Reply:
Updated to take a single edit